Headers/AppKit/NSCollectionView.h

```diff
@@ -34,9 +37,12 @@
 #import <AppKit/NSDragging.h>
 #import <AppKit/NSNibDeclarations.h>
 #import <AppKit/NSView.h>
+#import <AppKit/NSPasteboard.h>
```

Comment:
Wouldn't it be sufficient to declare the class here?

Reply:
I added forward declarations and removd the import.